paddle/fluid/operators/mkldnn/interpolate_mkldnn_op.cc

```diff
@@ -105,7 +105,8 @@ class InterpolateMKLDNNKernel : public framework::OpKernel<T> {
           }
         }
       }
-      if (scale[0] > 0.0f && scale[1] > 0.0f && scale[2] > 0.0f) {
+      if (scale.size() == 3 && scale[0] > 0.0f && scale[1] > 0.0f &&
+          scale[2] > 0.0f) {
```

Comment:
scale最多只能为3是吗

Reply:
是的

Reply:
Is there possibility that scale is less than 3? If its size is always 3, maybe we should set it when we define the vector on lin 86

Reply:
It is indeed possible that it is less than 3, and it may be 0, so I added a judgment here, and only make the subsequent judgment when it is equal to 3